plugins/content/config/model.schema

```diff
@@ -125,7 +125,7 @@
           "type": "string",
           "required": true,
           "default": "easeOutQuart",
-          "title": "Hide Easing",
```

Comment:
Should these be the other way round? We have easeOutQuart for the '_showEasing' animation.